.github/ISSUE_TEMPLATE/spike.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Spike
+about: Create a task to research or triage an issue that lacks sufficient clarity.
+title: '[ spike ] '
+labels: ''
+assignees: ''
+
+---
+
+## Description
+
+<!-- A clear and concise description of the issue or request that requires -->
+<!-- triage before it can be worked on. -->
+<!-- The sections suggested are intended to make it easy to create a -->
+<!-- descriptive issue Change as needed! -->
+
+## Questions / Areas of Uncertainty
+
+<!-- A list of unknowns for which we need information in order to fully -->
+<!-- understand the scope of the problem. -->
+
+## Outcome
+
+<!-- To be filled-in after all research has taken place. -->
+<!-- This may be an explanation of the issue or approach to be taken, -->
+<!-- a link to a newly-created Epic milestone, a new task/bug ticket, -->
```

Comment:
```suggestion
<!-- a link to a newly-created milestone, a new task/bug ticket, -->
```

Reply:
Sure, I can see how this is unnecessary. Adding it to a batch of suggestions. Thanks!